docs/installation_guide/installation.rst

```diff
@@ -183,6 +183,8 @@ Here’s the list of the singer connectors and if they are installed by default
 +----------------------------+---------------------------------------------+----------------------------------+---------------------------------------+
 | tap-mixpanel               | ./install --connectors=tap-mixpanel         | YES                              |                                       |
 +----------------------------+---------------------------------------------+----------------------------------+---------------------------------------+
+| tap-twilio                 | ./install --connectors=tap-twilio           | YES                              |                                       |
```

Comment:
How is it decided if it should be included in the default install?

Reply:
I think if we use it at TW then it's it's default install.

Reply:
Then I guess YES should be good here, thanks!